src/scores/emerging/risk_matrix.py

```diff
@@ -398,7 +396,8 @@ def weights_from_warning_scaling(
 
     References:
         - Taggart, R. J., & Wilke, D. J. (2025). Warnings based on risk matrices: a coherent framework
-          with consistent evaluation. arXiv. https://doi.org/10.48550/arXiv.2502.08891
+          with consistent evaluation. Nat. Hazards Earth Syst. Sci., 25, 2657–2677.
```

Comment:
```suggestion
          with consistent evaluation. Natural Hazards and Earth System Sciences, 25(8), 2657–2677.
```
As above - suggest adding full journal title & issue number